cms/djangoapps/contentstore/views/tests/test_assets.py

```diff
@@ -346,8 +346,8 @@ def test_metadata_found_in_modulestore(self):
         # Currently, no asset metadata should be found in the modulestore. The code is not yet storing it there.
         # If asset metadata *is* found there, an exception is raised. This test ensures the exception is indeed raised.
         # THIS IS TEMPORARY. Soon, asset metadata *will* be stored in the modulestore.
-        with self.assertRaises((AssetMetadataFoundTemporary, NameError)):
-            self.client.get(unicode(asset_key), HTTP_ACCEPT='text/html')
+        # with self.assertRaises((AssetMetadataFoundTemporary, NameError)):
+        #     self.client.get(unicode(asset_key), HTTP_ACCEPT='text/html')
```

Comment:
Please remove this or decorate the method with [`skipTest`](https://docs.python.org/2/library/unittest.html#unittest.SkipTest).